app/src/main/java/com/zulip/android/activities/MessageListFragment.java

```diff
@@ -290,6 +305,88 @@ private void selectPointer() {
         }
     }
 
+    /**
+     * Edit a message passed as parameter
+     *
+     * @param message Message to be edited
+     */
+    private void editMessage(final Message message, final int position) {
+        boolean isEditingAllowed = app.
+                getSettings().
+                getBoolean(Constants.IS_EDITING_ALLOWED, Constants.DEFAULT_EDITING_ALLOWED);
+        int maxMessageContentEditLimit = app.
```

Comment:
If isEditingAllowed is false, then there's no need of getting the time interval from settings, straight away show the toast and return!